mindmeld/ser.py

```diff
@@ -126,6 +130,15 @@ def parse_numerics(sentence, dimensions=None, language='EN', locale='en_US',
             * response_code (int): http status code.
     """
     if sentence == '':
+        logger.error('Empty query passed to the system entity resolver')
+        return {}, SUCCESSFUL_HTTP_CODE
+
+    if not language:
+        language = 'EN'
+
+    language = language.upper()
+    if len(language) != 2:
+        logger.error('System entity resolver only supports ISO 639-1 language codes')
         return {}, SUCCESSFUL_HTTP_CODE
```

Comment:
If the app passes in a 3 character language code, do we ever translate to the 2 character code, or does it run into this error?

Reply:
resolved